visma/io/tokenize.py

```diff
@@ -188,7 +188,15 @@ def tokenizeSymbols(terms):
     for i, term in enumerate(terms):
         symTokens.append('')
         if term in symbols:
-            if term == '*' or term == '/':
+            if term == '^':
+                if i + 1 < len(terms):
```

Comment:
```suggestion
                if i + 1 < len(terms)  and not isVariable(terms[i - 1]):
                    symTokens[-1] = 'Binary'
                else:
                    symTokens[-1] = False
```
aka short circuiting.